examples/rtc_s1_sample_runconfig-v2.0.0-er.4.0.yaml

```diff
@@ -129,13 +129,18 @@ RunConfig:
                         orbit_file_path:
                             - /home/rtc_user/input_dir/S1B_OPER_AUX_POEORB_OPOD_20180524T110543_V20180503T225942_20180505T005942.EOF
                         # Burst to process. Empty to process all bursts
-                        burst_id:
+                        burst_id: []
 
                     # This section should match the DynamicAncillaryFilesGroup of the PGE RunConfig
                     dynamic_ancillary_file_group:
                         # Digital elevation model
                         dem_file: /home/rtc_user/input_dir/dem.tif
 
+                    # This section should match the StaticAncillaryFilesGroup of the PGE RunConfig
+                    static_ancillary_file_group:
+                        # burst database sqlite file
+                        burst_database_file: /home/compass_user/input_dir/burst_database.sqlite3
```

Comment:
Small cut and paste issue here, this should be `/home/rtc_user/input_dir/burst_database.sqlite3`